config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1289,30 +1289,22 @@ presubmits:
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220928-cd48f52a16-master
+        command:
+        - runner.sh
         args:
-        - --root=/go/src
-        - "--job=$(JOB_NAME)"
-        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--service-account=/etc/service-account/service-account.json"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--timeout=240"
-        - --scenario=kubernetes_e2e
-        - -- # end bootstrap args, scenario args below
-        - --deployment=node
-        - --env=KUBE_SSH_USER=core
-        - --env=KUBE_FEATURE_GATES=DynamicResourceAllocation=true
-        - --gcp-zone=us-west1-b
-        - '--test_args=--nodes=4 --focus="\[Feature:DynamicResourceAllocation\]"'
-        - --provider=gce
-        - --timeout=180m
-        - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv1-serial.yaml
+        - /bin/sh
+        - -c
+        - >
+          make WHAT=github.com/onsi/ginkgo/v2/ginkgo &&
+          make WHAT=k8s.io/kubernetes/test/e2e/e2e.test &&
+          go install sigs.k8s.io/kind@main &&
+          test/e2e/dra/kind-build-image.sh dra/node:latest &&
+          kind create cluster --config test/e2e/dra/kind.yaml --image dra/node:latest &&
+          KUBECONFIG=${HOME}/.kube/config GINKGO_PARALLEL_NODES=8 E2E_REPORT_DIR=${ARTIFACTS} hack/ginkgo-e2e.sh -ginkgo.focus=DynamicResourceAllocation
         resources:
-          limits:
-            cpu: 4
-            memory: 6Gi
           requests:
-            cpu: 4
-            memory: 6Gi
```

Comment:
we have a policy to ensure limits and requests are present if the prowjob need to run on `cluster: k8s-infra-prow-build`.

Reply:
That came from some other job. I don't think I need it, do it? The kubernetes-csi jobs all don't have it, so I have also dropped it here.

Reply:
k8s-csi jobs are running on the default build cluster. resources definition is not required to make them run there. All the jobs with `cluster: k8s-infra-prow-build` have this resources def. It's a requirement established a long time ago (can't find the announcement for it).

Reply:
I had to fix also something else. Now all tests are passing and the job is very similar to [some other job](https://github.com/kubernetes/test-infra/blob/43c5445b6733d639dadaa46d923109524cc29613/config/jobs/kubernetes-csi/external-provisioner/external-provisioner-config.yaml#L51-L95) where I am using kind.

@ameukam please take another look.